kustomize/kustomization.yaml

```diff
@@ -19,5 +19,6 @@ resources:
   - base
 
 components:
-  - components/cymbal-branding
-  - components/google-cloud-operations
+  # - components/cymbal-branding
+  # - components/google-cloud-operations
+  # - components/memorystore
```

Comment:
Praise: I like that you've commented out all components by default. And you've included all components (in alphabetical order). 👌